tests/test_text_processing.py

```diff
@@ -166,6 +169,16 @@ def test_phoneme_to_sequence_with_blank_token():
     print(len(sequence))
     assert text_hat == text_hat_with_params == gt
 
+
+def assert_equal(actual: Any, expected: Any) -> None:
+    assert actual == expected, f"\n{actual} \n vs \n{expected}"
+
+
+def test_expand_numbers():
+    assert_equal("minus one", phoneme_cleaners("-1"))
```

Comment:
I just learned that pytest already handles asserts in a meaningful way: https://docs.pytest.org/en/2.1.0/assert.html